Bloom VS2012.sln

```diff
@@ -26,57 +26,81 @@ Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
 		Debug|Mixed Platforms = Debug|Mixed Platforms
+		Debug|x86 = Debug|x86
```

Comment:
Things didn't work right until I got both Bloom and the external program to explicitly compile for x86.  As part of that effort, VS 2012 helpfully (?) added all these changes to the .csproj and .sln files.